### ISSUE TITLE
Minor line break issue

### ISSUE THREAD

**Issue by adamzu** [cs50/submit50#13]
Super minor but if you type in the wrong username/password, the program tries to print a two line error message, but the `\n` gets interpreted literally: 
```
Sorry, something's wrong!\nLet sysadmins@cs50.harvard.edu know!
```

I think this is just an `echo` issue. I'm pretty sure `echo` doesn't really like line breaks, so maybe use `printf` instead to fix this?

**Comment by kzidane** [comment]
Fixed and PR'd per #14!

**Comment by dmalan** [comment]
Thanks, @adamzu!
